debezium-connector-mysql/src/test/java/io/debezium/relational/history/S3DatabaseHistoryTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational.history;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.entity.ContentType;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.adobe.testing.s3mock.testcontainers.S3MockContainer;
+
+import io.debezium.config.Configuration;
+import io.debezium.document.DocumentReader;
+import io.debezium.storage.s3.history.S3DatabaseHistory;
+
+import software.amazon.awssdk.auth.credentials.AnonymousCredentialsProvider;
+import software.amazon.awssdk.core.sync.ResponseTransformer;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.Bucket;
+import software.amazon.awssdk.services.s3.model.DeleteBucketRequest;
+import software.amazon.awssdk.services.s3.model.DeleteObjectRequest;
+import software.amazon.awssdk.services.s3.model.GetObjectRequest;
+import software.amazon.awssdk.services.s3.model.ListObjectsRequest;
+import software.amazon.awssdk.services.s3.model.S3Object;
+
+public class S3DatabaseHistoryTest extends AbstractDatabaseHistoryTest {
+    final public static String IMAGE_TAG = "2.4.14";
```

Comment:
I would adjust the MySQL pom to pass the `version.s3mock` property into the test suite as a system property and then use `System#getProperty` to resolve this value rather than using a hard-coded string here.  This has the advantage that we adjust the version reference in one spot and that if multiple tests are written across multiple connectors, we again source that version value from one logical place.